tutorials/single-page-tutorials/nested-nfts-with-wallet-lib.md

```diff
@@ -0,0 +1,423 @@
+# Create nested NFTs with the Wallet Library
+
+In this tutorial you will create three NFTs on the Shimmer testnet. On Shimmer, NFTs are not just dumb tokens, instead they have their own unique addresses and are able to own other NFTs. This concept is known as nested NFTs. In order to illustrate this concept, you will follow along the storyline below.
+
+## Storyline
+
+Let's imagine a game with avatars and items, which all are represented as unique NFTs on the Shimmer network. In this example it allows an avatar to not only hold items in-game, but its avatar NFT to also own its unique item NFTs directly on-DLT, in order to represent the dependencies between in-game components on the Shimmer network.
+
+![Blue Warrior](./images/warrior-blue.gif)
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/en/).
+- This tutorial will build on the tutorial [Testnet Wallet Setup](https://wiki.iota.org/tutorials/wallet-setup). If you haven't completed it, please do so, and afterward, come back here, or make sure you have a prefunded wallet. You will use the same folder and development environment as in the mentioned prerequired tutorial, allowing you to reuse the Stronghold account with funds on it.
```

Comment:
I'd actually prefer this to be more decoupled. The basic requirement is to have a funded wallet right? So my suggestion would be to provide alternatives here on how to get one, and just point to the wallet tutorial as one of the alternatives.

Reply:
Depends on how you define wallet. In this case the subsequent tutorials require a Stronghold account within the same folder in order to work. So just a mnemonic with funds on it is not sufficient.